src/reference/asciidoc/logging.adoc

```diff
@@ -235,7 +244,7 @@ In addition they populate headers with the following values:
 * The level of the log event
 * `thread`: the name of the thread where log event happened
 * The location of the stack trace of the log event call
-* A copy of all the MDC properties
+* A copy of all the MDC properties (in the case if `addMdcAsHeaders` is set to `true`)
```

Comment:
We need to state in the `whats-new.adoc` that now `MDC` is not copied into headers by default.
If we agree that it has to be `false` though...

Reply:
ok, will add it once we will make the final decision

Reply:
OK. So, final decision is to have it as `true` by default. That's it.
Thanks